examples/datasource-http/package.json

```diff
@@ -3,10 +3,15 @@
   "version": "1.0.0",
   "description": "An example plugin for Grafana",
   "scripts": {
-    "build": "grafana-toolkit plugin:build",
-    "test": "grafana-toolkit plugin:test",
-    "dev": "grafana-toolkit plugin:dev",
-    "watch": "grafana-toolkit plugin:dev --watch"
+    "build": "TS_NODE_PROJECT=\"./.config/webpack/tsconfig.webpack.json\" webpack -c ./.config/webpack/webpack.config.ts --env production",
+    "test": "jest --watch --onlyChanged",
+    "dev": "TS_NODE_PROJECT=\"./.config/webpack/tsconfig.webpack.json\" webpack -w -c ./.config/webpack/webpack.config.ts --env development",
+    "watch": "grafana-toolkit plugin:dev --watch",
```

Comment:
I think this can be removed in favor of the `dev` command since the `-w` flag will make webpack watch for changes. Right?